README.md

```diff
@@ -193,46 +201,55 @@ and [create a simulation job](https://docs.aws.amazon.com/robomaker/latest/dg/cr
 
 Procedurally generate an occupancy map for any gazebo world. This map can then be plugged in to navigate a robot in Worldforge worlds. For other aws-robotics worlds, this procedure is optional for the use-cases mentioned in this README. 
 
-### Pre-build
+### Script for default worlds/config
 
-```bash
-# Install dependencies (Ubuntu >18.04)
-sudo apt-get install ruby-dev libxml-xpath-perl libxml2-utils
-```
+Currently the following worlds are supported:
```

Comment:
Can be renamed to: Currently the following aws-robotics worlds are supported

Reply:
Fixed